docs/modules/ROOT/pages/install.adoc

```diff
@@ -0,0 +1,69 @@
+= Install
+
+You can install the extension from your browser's web-store, or a local copy of the extension app.
+
+== Install from web-store
+
+To install the Extension from your browser's web-store:
```

Comment:
The sentence might be a bit convoluted. I prefer direct and short sentences:

```suggestion
Install the Extension on your browser using the image links below:
```

And maybe we can add a `TIP: Alternatively, you can search "Asciidoctor.js Live Preview" in your Browser's Web Store.`

What do you think?